src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginAction.java

```diff
@@ -49,14 +49,12 @@ private EnvInjectVarList getEnvInjectVarList() {
                 new Maps.EntryTransformer<String, String, String>() {
                     public String transformEntry(String key, String value) {
                         final Set<String> sensibleVars = getSensibleVariables();
-                        return sensibleVars != null && sensibleVars.contains(key) ? "********" : value;
+                        return ((sensibleVars != null) && sensibleVars.contains(key)) ? EnvInjectPlugin.DEFAULT_MASK : value;
                     }
                 }));
     }
 
     public void buildEnvVars(AbstractBuild<?, ?> build, EnvVars env) {
-        final EnvInjectVarList varList = getEnvInjectVarList();
-        Map<String, String> envMap = varList.getEnvMap();
```

Comment:
What?Seems you're getting rid of these vars at all? what do you inject in next clauses